src/NBitcoin/Script.cs

```diff
@@ -820,7 +820,7 @@ private static BitcoinStream CreateHashWriter(HashVersion version)
             return a == null ? new Script(ops) : new Script(a._Script.Concat(new Script(ops)._Script));
         }
 
-        public IEnumerable<Op> ToOps()
+        public IList<Op> ToOps()
```

Comment:
Why? (just wondering)

Reply:
Because IList extends IEnumerable and inside of the method we already have the list. 

At some point I needed a list of opcodes so instead of ToOps().ToList() with this change I can just take the return value